server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/DomainMappingsRoutesTest.java

```diff
@@ -108,8 +107,7 @@ void addDomainMappingShouldRespondWithNoContent() {
             .when()
                 .put("from.com")
             .then()
-                .statusCode(HttpStatus.NO_CONTENT_204)
-                .body(isEmptyString());
```

Comment:
nothing prevent an HTTP server to say that there's no body and return some. This check makes sense